letsdrill.rb

```diff
@@ -1,18 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+ 	#Put your code here!
+  case
+  when integer <= 100 && integer >=90
+  	"A"
+  when integer >= 80
+  	"B"
+  when integer >= 70
+  	"C"
+  when integer >= 60
+  	"D"
+  when integer >= 0
+  	"F" 
+  else 
+  	"That's not a real score"
+  end
```

Comment:
Standard ruby styling is to indent two spaces.  ...take that for whatever you think it's worth.